<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Genosse Einhorn’s CD Audio Extractor</string>
  </property>
  <property name="windowIcon">
   <iconset resource="data/data.qrc">
    <normaloff>:/logo.svg</normaloff>:/logo.svg</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="hlToolbar">
      <item>
       <widget class="QToolButton" name="tbRefresh">
        <property name="toolTip">
         <string>Read the track list from the CD again.&lt;br/&gt;Useful if you changed the disc and the program didn't notice</string>
        </property>
        <property name="text">
         <string>&amp;Refresh</string>
        </property>
        <property name="icon">
         <iconset theme="view-refresh" resource="data/data.qrc">
          <normaloff>:/view-refresh.svg</normaloff>:/view-refresh.svg</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbLoadMetadata">
        <property name="toolTip">
         <string>Load Metadata from the Internet using the MusicBrainz Service</string>
        </property>
        <property name="text">
         <string>MusicBrainz</string>
        </property>
        <property name="icon">
         <iconset resource="data/data.qrc">
          <normaloff>:/musicbrainz.svg</normaloff>:/musicbrainz.svg</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbExtract">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Extract the selected tracks to audio files on your computer.&lt;/p&gt;&lt;p&gt;You'll set the file location and file type in the next step.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>&amp;Extract...</string>
        </property>
        <property name="icon">
         <iconset theme="document-save" resource="data/data.qrc">
          <normaloff>:/document-save.svg</normaloff>:/document-save.svg</iconset>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="spacer2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="tbMore">
        <property name="text">
         <string>&amp;More ▼</string>
        </property>
        <property name="icon">
         <iconset theme="help-browser" resource="data/data.qrc">
          <normaloff>:/help-browser.svg</normaloff>:/help-browser.svg</iconset>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
        <property name="autoRaise">
         <bool>true</bool>
        </property>
        <property name="arrowType">
         <enum>Qt::NoArrow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="metadataPage">
       <layout class="QVBoxLayout" name="verticalLayout1">
        <item>
         <widget class="QWidget" name="metadataWidget" native="true">
          <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,2,0,0,0,0">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item row="3" column="4">
            <widget class="QLineEdit" name="eYear">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>XXX</string>
             </property>
             <property name="placeholderText">
              <string>Unknown Year</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2" colspan="5">
            <widget class="QLineEdit" name="eComposer">
             <property name="text">
              <string>XXX</string>
             </property>
             <property name="placeholderText">
              <string>Unknown Composer</string>
             </property>
            </widget>
           </item>
           <item row="3" column="6">
            <widget class="QLineEdit" name="eDiscNo">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>XXX</string>
             </property>
             <property name="placeholderText">
              <string>1</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" colspan="5">
            <widget class="QLineEdit" name="eTitle">
             <property name="text">
              <string>XXX</string>
             </property>
             <property name="placeholderText">
              <string>Unknown Title</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="ClickableBuddyLabel" name="lTitle">
             <property name="text">
              <string>&amp;Title</string>
             </property>
             <property name="buddy">
              <cstring>eTitle</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="2" colspan="5">
            <widget class="QLineEdit" name="eArtist">
             <property name="text">
              <string>XXX</string>
             </property>
             <property name="placeholderText">
              <string>Unknown Artist</string>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="eGenre">
             <property name="text">
              <string>XXX</string>
             </property>
             <property name="placeholderText">
              <string>Unknown Genre</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="ClickableBuddyLabel" name="lGenre">
             <property name="text">
              <string>&amp;Genre</string>
             </property>
             <property name="buddy">
              <cstring>eGenre</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="ClickableBuddyLabel" name="lYear">
             <property name="text">
              <string>&amp;Year</string>
             </property>
             <property name="buddy">
              <cstring>eYear</cstring>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="ClickableBuddyLabel" name="lArtist">
             <property name="text">
              <string>&amp;Artist</string>
             </property>
             <property name="buddy">
              <cstring>eArtist</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="ClickableBuddyLabel" name="lComposer">
             <property name="text">
              <string>C&amp;omposer</string>
             </property>
             <property name="buddy">
              <cstring>eComposer</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="5">
            <widget class="ClickableBuddyLabel" name="lDiscNo">
             <property name="text">
              <string>&amp;CD</string>
             </property>
             <property name="buddy">
              <cstring>eDiscNo</cstring>
             </property>
            </widget>
           </item>
           <item row="0" column="0" rowspan="4">
            <widget class="CoverArtWidget" name="coverArt" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>150</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tvTracks">
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="nocdPage">
       <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>No Audio CD found :(</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>30</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="IconWidget" name="iwNoCdLogo" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="startPage">
       <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,0">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="IconWidget" name="iwStartLogo" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lStartProgramName">
          <property name="text">
           <string>CDDA-X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>30</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ClickableBuddyLabel</class>
   <extends>QLabel</extends>
   <header>uiutil/clickablebuddylabel.h</header>
  </customwidget>
  <customwidget>
   <class>CoverArtWidget</class>
   <extends>QWidget</extends>
   <header>coverartwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>IconWidget</class>
   <extends>QWidget</extends>
   <header>uiutil/iconwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tbRefresh</tabstop>
  <tabstop>tbExtract</tabstop>
  <tabstop>tbMore</tabstop>
  <tabstop>eTitle</tabstop>
  <tabstop>eArtist</tabstop>
  <tabstop>eComposer</tabstop>
  <tabstop>eGenre</tabstop>
  <tabstop>eYear</tabstop>
  <tabstop>eDiscNo</tabstop>
  <tabstop>tvTracks</tabstop>
 </tabstops>
 <resources>
  <include location="data/data.qrc"/>
 </resources>
 <connections/>
</ui>
